src/components/homepage-features.tsx

```diff
@@ -103,7 +103,32 @@ export default function HomepageFeatures(): JSX.Element {
       <div className={clsx('container--small', styles.footerCta)}>
         <h2>Ready to get started with Sequelize?</h2>
         <p>
-          Transactions, migrations, strong typing, JSON querying, and more.
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/transactions/"
+          > Transactions
+          </Link>,
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/migrations/"
+          > migrations
+          </Link>,
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/typescript/"
+          > strong typing
+          </Link>,
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/other-data-types/#json-sqlite-mysql-mariadb-and-postgresql-only"
+          > JSON querying
+          </Link>,
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/hooks/"
+          > lifecycle events (hooks)
+          </Link>
```

Comment:
```suggestion
          </Link>,
```

Reply:
By the way, I'm calling yarn and the lock file is updated

Reply:
Sure I will rever

Reply:
> By the way, I'm calling `yarn` command and the lock file is updated

It shouldn't, yarn has no reason to update the lockfile if package.json has not changed. It's not happening on my end

Reply:
> > By the way, I'm calling `yarn` command and the lock file is updated
> 
> It shouldn't, yarn has no reason to update the lockfile if package.json has not changed. It's not happening on my end

It's happen to me so I assume my machine broke xD

Reply:
You can take a look to the changes there is no package,json changes too

Reply:
Or..  it's happen because I'm calling the `npm install` command

Reply:
That's probably why
Revert it, delete your node_modules, then call `yarn` and you should end up with the proper state :)

Reply:
> That's probably why Revert it, delete your node_modules, then call `yarn` and you should end up with the proper state :)

Cool, thanks